chainerx/_docs/routines.py

```diff
@@ -607,6 +607,21 @@ def _docs_linalg():
 .. seealso:: :func:`numpy.linalg.inv`
 """)
 
+    _docs.set_doc(
+        chainerx.linalg.cholesky,
+        """cholesky(a)
+Returns the Cholesky decomposition for the square matrix ``a``.
```

Comment:
Not a strong opinion but how about being more explicit in what it returns, i.e. the L in L * L^{H,T}.

Reply:
Sure, I'll add a bit more info.